client/app/common/dashboard/service/loading.html

```diff
@@ -1,4 +1,4 @@
-<div class="loading text-center top-space-m20">
+<div class="loading text-center mt-20">
```

Comment:
⚠️ You need to set those classes with rem instead of px. Here's the consensus we have : 
2px   -> 1rem
4px   -> 2rem
8px   -> 2rem
12px -> 3rem
16px -> 3rem
20px -> 4rem
24px -> 4rem
32px -> 5rem